pyproject.toml

```diff
@@ -7,7 +7,7 @@ readme = "README.md"
 include = ["dpypelines/schemas/**/*.json"]
 
 [tool.poetry.dependencies]
-python = "3.9"
+python = ">=3.9.0,<3.10.0"
```

Comment:
I noticed that in order to run 'Make test' or any other command, I had to delete the existing 'poetry.lock' file because I encountered this error: 'pyproject.toml changed significantly since poetry.lock was last generated. Run poetry lock [--no-update] to fix the lock file.' 

The previous 'poetry.lock' file was configured for Python 3.9, while the updates included changes to the Python version specified in the 'pyproject.toml' file to '3.9.0,<3.10.0'.

Reply:
good spot 😄 , have regenerated the lock file